modules/manually-maintained-credentials-upgrade.adoc

```diff
@@ -8,20 +8,50 @@
 
-. Navigate to *Administration* -> *Cluster Settings*.
+* For z-stream releases, for example, from 4.7.12 to 4.7.13, the upgrade is not blocked, but the credentials must still be updated for the new release.
 
-. To view the CCO status details, click *cloud-credential* in the *Cluster Operators* list.
+Before upgrading a cluster with manually maintained credentials, you must create any new credentials for the release image that you are upgrading to. Additionally, you must review the required permissions for existing credentials and accommodate any new permissions requirements in the new release for those components.
 
-. If the *Upgradeable* status in the *Conditions* section is *False*, examine the `CredentialsRequest` custom resource for the new release and update the manually maintained credentials on your cluster to match before upgrading.
+The _Manually creating IAM_ section of the installation content for your cloud provider explains how to obtain and use the credentials required for your cloud.
 
-In addition to creating new credentials for the release image that you are
-upgrading to, you must review the required permissions for existing credentials
-and accommodate any new permissions requirements for existing components in the
-new release. The CCO cannot detect these mismatches and will not set
-`upgradable` to `false` in this case.
+.Procedure
 
-The _Manually creating IAM_ section of the installation content for your cloud provider explains how to obtain and use the credentials required for your cloud.
+. Examine the `CredentialsRequest` custom resource for the new release and update the manually maintained credentials on your cluster to match.
+
+. To indicate that the cluster is ready to upgrade:
+
+.. Log in to the {product-title} CLI as a user with the `cluster-admin` role.
+
+.. Edit the `CloudCredential` object to add an `upgradeable-to` annotation within the `metadata` field:
++
+[source,terminal]
+----
+$ oc edit cloudcredential cluster
+----
++
+.Text to add
++
+[source,yaml]
+----
+...
+  metadata:
+    annotations:
+      cloudcredential.openshift.io/upgradeable-to: <version_number>
+...
+----
++
+Where `<version_number>` is the version you are upgrading to, in the format `x.y.z`. For example, `4.7.0` for {product-title} 4.7.0.
+
+. Use the *Administrator* perspective of the web console to determine if the CCO is upgradeable. Note that it may take several minutes for the status to change.
+
+.. Navigate to *Administration* -> *Cluster Settings*.
+
+.. To view the CCO status details, click *cloud-credential* in the *Cluster Operators* list.
+
+.. If the *Upgradeable* status in the *Conditions* section is *False*, verify that the manually maintained credentials on your cluster match the updated `CredentialsRequest` custom resource for the new release.
+
+When the *Upgradeable* status in the *Conditions* section is *True*, you can begin the {product-title} upgrade.
```

Comment:
For this Procedure section I think we just need to move things around so the order makes more sense. Something like this:

STEP Extract the CredentialsRequest resources from the release image being upgraded to (just like the steps performed before installation).
STEP Create any new Secrets needed (for CredentialsRequest resources that are new to the release image being upgraded to).
STEP Update the permissions of any existing credentials stored in Secrets that already exist if the equivalent CredentialsRequest resources have changed their permissions requirements.

Once all the new Secrets are created, and any permissions changes for previous Secrets have been updated, it is time to signal that the cluster is clear for upgrade. To indicate that the cluster is ready to upgrade:

.. Log in to the {product-title} CLI as a user with the `cluster-admin` role.

.. Edit the `CloudCredential` object to add an `upgradeable-to` annotation within the `metadata` field:
+
[source,terminal]
----
$ oc edit cloudcredential cluster
----
+
.Text to add
+
[source,yaml]
----
...
  metadata:
    annotations:
      cloudcredential.openshift.io/upgradeable-to: <version_number>
...
----
+
Where `<version_number>` is the version you are upgrading to, in the format `x.y.z`. For example, `4.7.0` for {product-title} 4.7.0.

When the *Upgradeable* status in the *Conditions* section is *True*, you can begin the {product-title} upgrade.